healthcheck.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Automattic\VIP;
+
+use Automattic\VIP\Utils\Context;
+/**
+ * Utility Healthcheck class that runs further than cache-healthcheck and allow to verify the app code is working.
+ */
+class Healthcheck {
+	/**
+	 * By default the site is considered healthy unless filter value returns falsy value
+	 *
+	 * @var boolean
+	 */
+	protected bool $healthy = true;
```

Comment:
personally I like one newline space after a property declaration but I noticed the linter didn't agree so :shrug: